registration/include/pcl/registration/impl/correspondence_estimation.hpp

```diff
@@ -112,6 +112,35 @@ CorrespondenceEstimationBase<PointSource, PointTarget, Scalar>::initComputeRecip
   return (true);
 }
 
+namespace detail {
+
+template <
+    typename PointSource,
+    typename PointTarget,
+    typename Index,
```

Comment:
If we put PointTarget as 1st argument, the rest can be auto-deduced based on function argument.

Reply:
done